src/app/rpmostree-override-builtins.cxx

```diff
@@ -120,17 +129,21 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
           g_autofree char *refresh_transaction_address = NULL;
 
           if (!rpmostree_os_call_refresh_md_sync (os_proxy, options, &refresh_transaction_address,
-                                                 cancellable, error))
+                                                  cancellable, error))
             return FALSE;
           if (!rpmostree_transaction_get_response_sync (sysroot_proxy, (const char*)refresh_transaction_address,
                                                        cancellable, error))
             return FALSE;
           if (!rpmostree_osexperimental_call_download_packages_sync (osexperimental_proxy,
                                                                      (const char *const*)queries->pdata,
-                                                                     NULL, &fetched_rpms_fds,
+                                                                     opt_from, NULL, &fetched_rpms_fds,
                                                                      cancellable, error))
             return FALSE;
-
+          if (!fetched_rpms_fds)
+            {
+              g_print ("No new version available for %s\n", (char*)queries->pdata[0]);
+	      return TRUE;
+            }
```

Comment:
This can't happen the way the API is designed since it always returns fds in the happy path.